instagram-app/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import dummyData from './dummy-data';
+
+import PostContainer from './components/PostContainer/PostContainer';
+import './App.css';
+
+class App extends React.Component {
+  constructor() {
```

Comment:
you don't actually need this constructor, or the super().
you can simply write it as

```
class App extends React.Component {
  state = {
      instaData: dummyData
    };
```